doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -3057,3 +3057,39 @@ Example of ARP logging: request and response
     "dest_mac": "00:1d:09:f0:92:ab",
     "dest_ip": "10.10.10.1"
   }
+
+Event type: POP3
+----------------
+
+Fields
+~~~~~~
+
+* "command" (optional): a request (aka command) sent by the pop3 client
+  * "command.keyword" (string): pop3 keyword, if unknown but valid `error_flags` will be have `UnknownKeyword` flag set
+  * "command.args" (array of strings)
+* "response": a response sent by the pop3 server
```

Comment:
also optional, right?

Reply:
Kind of, only in the case of a MAX_TX error.